packages/tiled/lib/src/common/color.dart

```diff
@@ -0,0 +1,25 @@
+part of tiled;
+
+/// Basic data class holding a Color in ARGB format.
+/// This can be converted to dart:ui's Color using the flame_tiled package
+class ColorData {
+  static int _sub(int hex, int index) => (hex >> index * 8) & 0x000000ff;
```

Comment:
I just saw this in the other PR; is there a mix up between the two? If so; my comment there is valid here as well: why not store the "hex" as one value and do bit shifting in setters?

This is what `Color` was doing before and so the pattern would match and the memory footprint should be the same.

Reply:
Yes you are right that would be more efficient! This change does belong here, but I based the PR‘s branch on this one so I don‘t have to integrate the new types later!